frontend/components/CompactNavigationBar.js

```diff
@@ -0,0 +1,440 @@
+            ) : (
+              <MenuIcon className={styles.menuIcon} />
+            )}
+          </button>
+        </div>
+      </div>
+      {/* make menu visible if the menu is open */}
+      <div
+        className={
+          menuOpen
+            ? styles.menuContainer
+            : `${styles.hiddenMenu} ${styles.menuContainer}`
+        }
+      >
+        {/* if the navbar is transparent then the menu is also transparent,
+            same structure for the other menu links */}
+        <div
+          className={
+            clearNavBar && router.pathname != "/404"
+              ? `${styles.clearMenuContent} ${styles.menuContent}`
+              : styles.menuContent
+          }
+        >
+          <Link href="/">
+            <div className={styles.menuItemContent}>
+              {/* underline current page name in the menu, 
+                  same structure for the other links */}
+              <div
+                className={
+                  router.asPath === "/"
+                    ? `${styles.currentPageMenuUnderline} ${styles.menuItem}`
+                    : styles.menuItem
+                }
+                onClick={menuItemClick}
+              >
+                <a>Home</a>
+              </div>
+            </div>
+          </Link>
+          {/* div for dropdown inside menu */}
+          <div
+            className={styles.menuDropdownContainer}
+            id="aboutUsDropdown"
+            onClick={aboutUsDropdownClick}
+          >
+            {/* apply dropdown underline and red text color accordingly:
+                if the current page belongs to the dropdown, the text should be underlined
+                if the dropdown is open, the text should be red
+                same structure for the other dropdowns */}
+            <div
+              className={
+                router.asPath.split("/")[1] === "about" && aboutUsDropdownOpen
+                  ? `${styles.currentPageMenuUnderline} ${styles.currentDropdown} ${styles.menuItem}`
+                  : router.asPath.split("/")[1] === "about"
+                  ? `${styles.currentPageMenuUnderline} ${styles.menuItem}`
+                  : aboutUsDropdownOpen
+                  ? `${styles.currentDropdown} ${styles.menuItem}`
+                  : styles.menuItem
+              }
+            >
+              <a id="aboutUsText">About Us ▸</a>
+            </div>
+          </div>
+          {/* make dropdown visible if the dropdown is open */}
+          <div
+            className={
+              aboutUsDropdownOpen
+                ? styles.menuDropdownContent
+                : `${styles.menuDropdownContent} ${styles.hiddenMenuDropdownContent}`
+            }
+          >
+            <Link href="/about/our-story">
+              {/* make the current page name in the dropdown bold,
+                  same structure for the other dropdown links */}
+              <div
+                className={
```

Comment:
Use the `map` function to iterate through the details of the dropdown rather than having it hardcoded throughout. Take a look at Footer.js for reference.